tokio/src/runtime/io/scheduled_io.rs

```diff
@@ -17,6 +17,93 @@ use std::task::{Context, Poll, Waker};
 
 /// Stored in the I/O driver resource slab.
 #[derive(Debug)]
+// # This struct should be cache padded to avoid false sharing. The cache padding rules are copied
```

Comment:
Yes, I know it is time to factor this out into a CachePadded abstraction. I pinky promise I'll do it in a follow-up, but I don't want to mix up concerns in this PR.